frontend/src/components/layout/Header.tsx

```diff
@@ -248,19 +259,27 @@ const Header = () => {
                       <div>
                         <Menu.Button className="flex rounded-full bg-white">
                           <span className="sr-only">{t("open-user-menu")}</span>
-                          <Image
-                            className="rounded-full"
-                            src={
-                              Object.values(user.info.auths).find(
-                                (auth) => auth.avatar,
-                              ).avatar
-                            }
-                            width="38"
-                            height="38"
-                            alt={t("user-avatar", {
-                              user: user.info.displayname,
-                            })}
-                          />
+                          {userAvatarUrl ? (
+                            <Image
+                              className="rounded-full"
+                              src={userAvatarUrl}
+                              width="38"
+                              height="38"
+                              alt={t("user-avatar", {
+                                user: user.info.displayname,
+                              })}
+                            />
+                          ) : (
+                            <Avatar
+                              size={40}
+                              name={user.info.displayname}
+                              variant="marble"
+                              colors={[
+                                "rgb(74, 144, 217)",
```

Comment:
I guess these are some of our colors (can't tell from the setup I'm on right now)

Reply:
Yes, it is. I used hardcoded values because I had issues accessing them from JS.

I'm not really sure how I would do that, as all Tailwind theme colors are used as component props and not extracted out as JS variables in any instance I could find :(

Reply:
I don't know either, but using the hex values might at least make it easier to recognize the first one?